docs/api/python/autograd/autograd.md

```diff
@@ -76,7 +82,63 @@ Detailed tutorials are available in Part 1 of
 [the MXNet gluon book](http://gluon.mxnet.io/).
 
 
+# Higher order gradient
+
+Some operators support higher order gradients. Meaning that you calculate the gradient of the
+gradient. For this the operator's backward must be differentiable as well. Some operators support
+differentiating multiple times, and others two, most just once.
+
+For calculating higher order gradients, we can use the `mx.autograd.grad` function while recording
```

Comment:
They could just call backward() instead of `mx.autograd.grad`, right?

Reply:
I think is consistent to call grad two times and more explicit specially when using HO gradients. But that's just my personal opinion. Shall I add that you can call backward?

Reply:
Resolving as per below.